report/src/report.cpp

```diff
@@ -190,7 +190,7 @@ auto compose_message(const sc_report& rep, const scp::LogConfig& cfg)
         }
         if (*rep.get_msg())
             os << rep.get_msg();
-        if (rep.get_severity() > SC_INFO) {
+        if (rep.get_severity() >= cfg.file_info_from) {
```

Comment:
Using >= here and the default for file_info_from being sc_core::SC_INFO does not resemble the original behavior. Is this intended?

Reply:
That's an error - I prefer the >= I think it makes more sense, but I intended to leave the thing as it was as a default. I'll fix.

Reply:
Fixed I believe

Reply:
Yes, indeed.